README.rst

```diff
@@ -73,45 +93,77 @@ Update your ``.travis.yml`` file with the following::
       - export DISPLAY=:99.0
       - sh -e /etc/init.d/xvfb start
```

Comment:
I think it doesn't work anymore.

see:

https://github.com/collective/collective.cover/pull/866
https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui